cmd/src/debug_comp.go

```diff
@@ -0,0 +1,228 @@
+package main
+
+import (
+	"archive/zip"
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+
+	"golang.org/x/sync/errgroup"
+
+	"golang.org/x/sync/semaphore"
+)
+
+func init() {
+	usage := `
+'src debug compose' invokes docker cli diagnostic commands targeting a containers that are members of a docker-compose network, 
+writing an archive file from their returns. 
+
+Usage:
+
+	src debug compose [command options]
+
+Flags:
+
+	-o					Specify the name of the output zip archive.
+	--no-configs		Don't include Sourcegraph configuration json.
+
+Examples:
+
+	$ src debug compose -o debug.zip
+
+	$ src -v debug compose -no-configs -o foo.zip
+
+`
+
+	flagSet := flag.NewFlagSet("compose", flag.ExitOnError)
+	var base string
+	var noConfigs bool
+	flagSet.StringVar(&base, "o", "debug.zip", "The name of the output zip archive")
+	flagSet.BoolVar(&noConfigs, "no-configs", false, "If true include Sourcegraph configuration files. Default value true.")
```

Comment:
The help text here says the default is `true`, but the default value in the `BoolVar` call is `false`.